Common/Models/LoopSettingsUserInfo.swift

```diff
@@ -6,10 +6,67 @@
 //
 
 import LoopCore
+import LoopKit
 
+struct LoopSettingsUserInfo: Equatable {
```

Comment:
This new struct can carry settings and override activations from and to the watch app. Override activations were moved out of settings.